.github/workflows/alpine.yml

```diff
@@ -0,0 +1,44 @@
+# Build dub on Alpine Linux, testing compatibility with Musl
+name: Alpine
+
+on:
+  pull_request:
+    branches:
+      - master
+      - stable
+  push:
+    branches:
+      - master
+      - stable
+      # Use this branch name in your fork to test changes
+      - github-actions
+
+jobs:
+  main:
+    name: Run
+    strategy:
+      # Default, disable if you want to debug
+      fail-fast: false
+      matrix:
+        include:
+# Disabled as we rely on DIP1000 `foreach (scope)` which GDC < 12 doesn't support
+#          - { dc: gdc, dcpkg: gcc-gdc, dcbin: gdc  }
```

Comment:
can we start using GDC 12 now?

Reply:
We use whatever Alpine packages. They usually update quite quickly so I expect we'll be able to do so quite soon.